### PR TITLE
add linting

### PR DESCRIPTION
**Pull request by danlaine**
Updates the linter settings to be the same as avalanchego uses.

@holisticode @felipemadero 
There are 2 TODOs in here that the linter is complaining about that I'm not sure how to fix. Please have a look at those.

### CODE REVIEW

File: local/snapshot.go [hunk 1]
```diff
@@ -122,20 +122,22 @@ func (ln *localNetwork) SaveSnapshot(ctx context.Context, snapshotName string) (
 		return "", err
 	}
 	// create main snapshot dirs
-	snapshotDbDir := filepath.Join(filepath.Join(snapshotDir, defaultDbSubdir))
+	// TODO for CLI team -- what is the line below supposed to do?
```
Comment (holisticode) [inline local/snapshot.go:125]: @felipemadero the first `Join` isn't doing anything, that's why the linter complains. What was the idea?

File: local/snapshot.go [hunk 2]
```diff
@@ -178,8 +180,10 @@ func (ln *localNetwork) loadSnapshot(
 ) error {
 	ln.lock.Lock()
 	defer ln.lock.Unlock()
+	// TODO for CLI team -- what are the lines below supposed to do?
+	// The linter is complaining about them.
 	snapshotDir := filepath.Join(ln.snapshotsDir, snapshotPrefix+snapshotName)
-	snapshotDbDir := filepath.Join(filepath.Join(snapshotDir, defaultDbSubdir))
+	snapshotDbDir := filepath.Join(filepath.Join(snapshotDir, defaultDBSubdir))
```
Comment (holisticode) [inline local/snapshot.go:186]: @felipemadero same here, the first `Join` is not doing anything.

**Review by holisticode — CHANGES_REQUESTED** [review]

File: local/snapshot.go (hunk 1 above)
Comment (felipemadero) [reply local/snapshot.go:125]: ohh this is just an error, just remove the outer join

File: local/snapshot.go (hunk 2 above)
Comment (felipemadero) [reply local/snapshot.go:186]: same

**Review by felipemadero — APPROVED** [review]

**Review by holisticode — APPROVED** [review]

**Pull request merged**